apps/src/music/constants.ts

```diff
@@ -70,3 +70,6 @@ export const DEFAULT_CHORD = {
 };
 
 export const DEFAULT_CHORD_LENGTH = 1;
+
+export const LOCAL_STORAGE = 'local';
+export const S3_STORAGE = 's3';
```

Comment:
One thought (here and elsewhere). Having the client know the difference between local and remote (server?) storage makes sense to me. However, having the client know that the backend specifically uses S3 as its storage technology seems a little strange. Ideally, the backend could switch to a different technology, cloud provider, or could even deterministically choose between multiple technologies without the client side knowing the difference.

Renaming to something like "remote" or "server" rather than "s3" could help ensure that level of encapsulation.

Reply:
good call, I will rename. This was not intended to be a long-term flag, just one so we could have both versions running in parallel